### PR TITLE
Remove duplicated nginx image. Use nginx-slim instead

### PR DESCRIPTION
**Pull request by aledbf**
This PR removes the image `gcr.io/google_containers/nginx:1.7.9` and uses `gcr.io/google_containers/nginx-slim:0.7`.
Besides removing the duplication `1.7.9` is 16 months old.

### CODE REVIEW

**Comment by aledbf** [comment]
ping @bprashanth

File: cluster/saltbase/salt/e2e-image-puller/e2e-image-puller.manifest [hunk 1]
```diff
@@ -27,7 +27,7 @@ spec:
...
...ogle_containers/mounttest:0.5 gcr.io/google_containers/mounttest:0.6 gcr.io/google_containers/mounttest-user:0.3 gcr.io/google_containers/netexec:1.4 gcr.io/google_containers/netexec:1.5 gcr.io/google_containers/nettest:1.7 gcr.io/google_containers/nettest:1.8 gcr.io/google_containers/nginx-slim:0.7 gcr.io/google_containers/n-way-http:1.0 gcr.io/google_containers/pause:2.0 gcr.io/google_containers/pause-amd64:3.0 gcr.io/google_containers/porter:cd5cb5791ebaa8641955f0e8c2a9bed669b1eaab gcr.io/google_containers/portforwardtester:1.0 gcr.io/google_containers/redis:e2e gcr.io/google_containers/resource_consumer:beta2 gcr.io/google_containers/serve_hostname:v1.4 gcr.io/google_containers/servicelb:0.1 gcr.io/google_containers/test-webserver:e2e gcr.io/google_containers/ubuntu:14.04 gcr.io/google_containers/update-demo:kitten gcr.io/google_containers/update-demo:nautilus gcr.io/google_containers/volume-ceph:0.1 gcr.io/google_containers/volume-gluster:0.2 gcr.io/google_containers/volume-iscsi:0.1 gcr.io/google_containers/volume-nfs:0.6 gcr.io/google_containers/volume-rbd:0.1 gcr.io/google_samples/gb-redisslave:v1; do echo $(date '+%X') pulling $i; docker pull $i 1>/dev/null; done; exit 0;"
```
Comment (bprashanth) [inline cluster/saltbase/salt/e2e-image-puller/e2e-image-puller.manifest]: can you also remove servicelb from this list? we don't run that e2e in a blocking suite so it's not worth pulling

**Comment by bprashanth** [comment]
Thanks for the cleanup!

File: cluster/saltbase/salt/e2e-image-puller/e2e-image-puller.manifest (hunk 1 above)
Comment (aledbf) [inline cluster/saltbase/salt/e2e-image-puller/e2e-image-puller.manifest]: done

**Comment by aledbf** [comment]
ping @bprashanth

**Comment by bprashanth** [comment]
@k8s-bot test this github issue: #27551

**Comment by bprashanth** [comment]
@k8s-bot test this github issue: #https://github.com/kubernetes/kubernetes/issues/27551

**Pull request merged**